settings.gradle.kts

```diff
@@ -135,6 +135,7 @@ dependencyResolutionManagement {
                 }
             }
         }
+        mavenLocal()
```

Comment:
Why do we need this change, please?

Reply:
flexa is not on maven, I am instead building their sdk locally lol